CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-19.5.3
+
+* CLI
+  * **New** Added a `linkerd edges` command that shows the source and
+    destination name and identity for proxied connections, to assist in
+    debugging
+  * Tap can now be disabled for specific pods during injection by using the
+    `--disable-tap` flag, or by using the `config.linkerd.io/disable-tap`
+    annotation
+  * Introduced pre-install healthcheck for clock skew (thanks, @matej-g!)
+* Controller
+  * Added Controller Component Labels to the webhook config resources (thanks,
+    @Pothulapati!)
+  * Moved the tap service into its own pod
+* Proxy
+  * Fix an epoll notification issue that could cause excessive CPU usage
+  * Allow disabling tap by setting an env var (thanks, @zaharidichev!)
```

Comment:
Can we phrase this so that it begins with "add", i.e. "Added the ability to disable tap by..."?

Reply:
Or "Add LINKERD2_PROXY_TAP_DISABLED env var which disable tap when set" if my prior suggestion was too passive voice?